blog/PyMC_Past_Present_Future.md

```diff
@@ -0,0 +1,166 @@
+(PyMC_past_present_future)=
+# PyMC: Past, Present, and Future (PyMCon 2020)
+
+:::{post} January 25, 2023
+:tags: events, community, talks
+:category: news, testimonial
+:author: Sangam SwadiK, Rowan Schaefer
+:image: 0
+:exclude:
+
+Summary of Chris Fonnesbeck's PyMCon 2020 keynote.
+:::
+
+
+- Table of Contents
+    1. Background
+    2. PyMC Development Timeline
+        - Early 2000s
+        - PyMC 1.0
+        - PyMC 2.0
+        - PyMC 3.0
+        - Post-3.0
+    3. Applications
+    4. Present Day
+    5. Contributing
```

Comment:
I would not add a table of contents as it already appears in the right sidebar with clickable links. Best case scenario with the manual one is info is repeated with little to no gain, worst case scenario, the manual one is wrong and confuses readers.

I would modify the headers and their hierarchy to look more like this. That is, all the elements in the numbered list should be level 2 headers. 

There currently is an extra level 1 "summary timeline" which I would remove completely and leave only the already existing "development timeline", and there doesn't seem to be any applications one. (I am not worried about the extra headers like what's next and connect with pymc though.